plugins/Installation/lang/en.json

```diff
@@ -137,6 +137,9 @@
         "SystemCheckPhpSetting": "To prevent some critical issue, you must set the following in your php.ini file: %s",
         "SystemCheckUpdateHttps": "Update over HTTPS",
         "SystemCheckUpdateHttpsNotSupported": "Matomo cannot use HTTPS to update, it will fall back to the insecure HTTP update. Check that CURL or allow_url_fopen is supported and that the openssl PHP extension is installed: https:\/\/matomo.org\/faq\/troubleshooting\/faq_177\/.",
+        "SystemCheckProcessorFormat" : "OS Format",
+        "SystemCheckProcessorFormat64": "64-bit",
+        "SystemCheckProcessorFormat32": "32-bit.<p> We highly recommend upgrading to a 64-bit to prevent various issues that may cause inconvenient in the future.</p>",
```

Comment:
I'm no native English speaker, so bare with me if I'm wrong, but shouldn't that be `inconvenience` instead of `inconvenient`? 🤔

Reply:
Maybe also write `to a 64-bit PHP binary`